src/components/NFTOriginVerifier.tsx

```diff
@@ -195,6 +211,56 @@ export default function NFTOriginVerifier() {
           }
         }
       })();
+    } else if (
+      lookupChain === CHAIN_ID_APTOS &&
+      lookupCreatorAddress &&
+      lookupCollectionName &&
+      lookupTokenName
+    ) {
+      (async () => {
+        try {
+          setIsLoading(true);
+          const tokenId: TokenTypes.TokenId = {
+            token_data_id: {
+              creator: lookupCreatorAddress,
+              collection: lookupCollectionName,
+              name: lookupTokenName,
+            },
+            property_version: "0",
+          };
+          const aptosClient = getAptosClient();
+          const tokenClient = new TokenClient(aptosClient);
+          const info = await getOriginalAssetAptos(
+            aptosClient,
+            getNFTBridgeAddressForChain(CHAIN_ID_APTOS),
+            tokenId
+          );
+          const tokenData = await tokenClient.getTokenData(
+            lookupCreatorAddress,
+            lookupCollectionName,
+            lookupTokenName
+          );
+          if (!cancelled) {
+            setIsLoading(false);
+            setParsedTokenAccount({
+              amount: "0",
+              decimals: 0,
+              mintKey: `${tokenData.collection} ${tokenData.name}`,
+              publicKey: "",
+              uiAmount: 0,
+              uiAmountString: "0",
+              uri: tokenData.uri,
```

Comment:
This is just a preference, using `destructuring`.

`const {collention, name, uri} = tokenData || {}`

```suggestion
          const tokenData = await tokenClient.getTokenData(
            lookupCreatorAddress,
            lookupCollectionName,
            lookupTokenName
          );
          const {collention, name, uri} = tokenData || {}
          
          if (!cancelled) {
            setIsLoading(false);
            setParsedTokenAccount({
              amount: "0",
              decimals: 0,
              mintKey: `${collection} ${name}`,
              publicKey: "",
              uiAmount: 0,
              uiAmountString: "0",
              uri,
```

Reply:
fixed